src/hooks/core/chat/useChatUtilsHook.ts

```diff
@@ -3,7 +3,7 @@ import SimpleToast from 'react-native-simple-toast';
 import moment from 'moment';
 import {atom, useRecoilState} from 'recoil';
 import {useNavigation, CommonActions} from '@react-navigation/native';
-
+import {Keyboard} from 'react-native';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of the `Keyboard` module from 'react-native' is a good addition to handle keyboard dismissal.
```diff
+ import {Keyboard} from 'react-native';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->